.drone/drone.jsonnet

```diff
@@ -1,6 +1,7 @@
 local supported_golang_versions = [
   '1.19.3',
```

Comment:
How many versions back should we test? Just wondering where to draw the line :)

Reply:
I think following [Go's release policy](https://go.dev/doc/devel/release#policy) makes sense in general. I've kept 1.19 for now as that's the minimum version `go.mod` declares.